### PR TITLE
Labster/teacher dashboard improvements

### PR DESCRIPTION
**Pull request by polesye**
WEB-119, WEB-121 @auraz please review.

### CODE REVIEW

File: openedx/core/djangoapps/labster/course/views.py [hunk 1]
```diff
@@ -149,8 +149,9 @@ def course_handler(request, course_key_string=None):
 @authentication_classes((SessionAuthentication, TokenAuthentication))
 @permission_classes((IsAuthenticated,))
 def coach_list_handler(reuqest):
-    coach_list = ({
-        'full_name': ccx.coach.get_full_name(),
-        'email': ccx.coach.email,
-    } for ccx in CustomCourseForEdX.objects.all())
-    return Response(coach_list)
+    coaches = set(ccx.coach for ccx in CustomCourseForEdX.objects.all())
```
Comment (auraz) [inline openedx/core/djangoapps/labster/course/views.py]: please add TODO comment to improve in the future to docstring

**Comment by auraz** [comment]
One small comment, otherwise :+1: 
We need to start thinking about adding tests :)

File: openedx/core/djangoapps/labster/course/views.py [hunk 2]
```diff
@@ -149,8 +150,12 @@ def course_handler(request, course_key_string=None):
...
 def coach_list_handler(reuqest):
-    coach_list = ({
-        'full_name': ccx.coach.get_full_name(),
-        'email': ccx.coach.email,
-    } for ccx in CustomCourseForEdX.objects.all())
-    return Response(coach_list)
+    """
+    Returns a list of dicts with information about CCX coaches (full name, email).
+    """
+    coaches = User.objects.filter(courseaccessrole__role=CourseCcxCoachRole.ROLE)
```
Comment (polesye) [inline openedx/core/djangoapps/labster/course/views.py:156]: @auraz please take a look. 
Previous solution with `set` is replaced with this line.

**Comment by auraz** [comment]
:+1:

**Pull request merged**